site/content/en/reference/live/apply/_index.md

```diff
@@ -36,6 +36,15 @@ group.  `kpt live apply` tracks the state of your applied resource set and
 related configuration. This helps `kpt` to reliably reconcile the real world
 resources with your configuration changes.
 
+### Client-Side Apply versus Server-Side Apply
+
+kpt live apply defaults to client-side apply, so the updates are accomplished
+by calculating and sending a patch from the client. Server-side apply
+with the `--server-side` flag sends the entire resource to the server
+for the update. The server-side flags and functionality are the same
+as kubectl. More information on
+[kubectl server-side apply]: <https://kubernetes.io/docs/reference/using-api/server-side-apply/>
```

Comment:
We usually put the links at the bottom of the page.

Reply:
Fixed.

Reply:
Added documentation in the `preview` command

Reply:
Resolved by adding documentation for these two additional flags.